src/platform/datapath_raw_socket.c

```diff
@@ -783,3 +783,328 @@ CxPlatFramingWriteHeaders(
     Buffer->Length += TransportLength + IpHeaderLen + sizeof(ETHERNET_HEADER);
     Buffer->Buffer -= TransportLength + IpHeaderLen + sizeof(ETHERNET_HEADER);
 }
+
+
+QUIC_STATUS
+CxPlatTryAddSocket(
```

Comment:
This stuff is generally fine, but eventually we're going to eliminate all this code, because we'll merge XDP support as a fast path to the normal (iocp, epoll) datapaths. So don't waste too much time/energe in getting this stuff just right because it will go away.

Reply:
OK